docs/docs/reference/crd-reference/config.md

```diff
@@ -39,17 +40,28 @@ spec:
       to put into the same auto-generated [KeptnApp](app.md).
       The default value is 30 (seconds).
     * **cloudEventsEndpoint** -- Endpoint where the lifecycle operator posts Cloud Events.
+    * **blockDeployment** -- An option used to block the deployment of the application until the
+      pre-deployment tasks and evaluations succeed.
```

Comment:
```suggestion
    * **blockDeployment** -- If set to `true` (default), application deployment is blocked until the
      pre-deployment tasks and evaluations succeed.
      You can set this field to `false` when building up
      your pre-deployment tasks and evaluations
      so that your application is deployed
      even if the pre-deployment tasks and/or evaluations fail.
```